<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="arrangementTab">
       <attribute name="title">
        <string>Arrangement</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="arrangementGraphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stereogramTab">
       <attribute name="title">
        <string>Stereogram</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="stereogramGraphicsView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="generateStereogramButton">
        <property name="text">
         <string>&amp;Generate stereogram</string>
        </property>
        <property name="shortcut">
         <string>Alt+G</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_LoadOnrakan"/>
    <addaction name="action_Save"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuStereogram">
    <property name="title">
     <string>Stereogram</string>
    </property>
    <widget class="QMenu" name="menuSet_tile">
     <property name="title">
      <string>Load tile</string>
     </property>
     <addaction name="action_LoadTileFromFile"/>
     <addaction name="action_LoadRandomDotTile"/>
    </widget>
    <addaction name="action_LoadDepthMap"/>
    <addaction name="menuSet_tile"/>
    <addaction name="action_Export"/>
    <addaction name="action_Options"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStereogram"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New onrakan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save onrakan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Onrakan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_LoadDepthMap">
   <property name="text">
    <string>Load &amp;depthmap</string>
   </property>
   <property name="toolTip">
    <string>Change depth-map image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="text">
    <string>Export stereogram</string>
   </property>
   <property name="toolTip">
    <string>Save the stereogram image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_LoadTileFromFile">
   <property name="text">
    <string>from file</string>
   </property>
   <property name="toolTip">
    <string>Load tile image from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_LoadRandomDotTile">
   <property name="text">
    <string>random dot</string>
   </property>
   <property name="toolTip">
    <string>Auto-generate a new random-dot image for use as the tile</string>
   </property>
  </action>
  <action name="action_LoadOnrakan">
   <property name="text">
    <string>&amp;Load onrakan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <resources/>
 <connections/>
</ui>
